### PR TITLE
🐛 moved US CPI fetching from API to snapshot step and changed dependencies

### PR DESCRIPTION
**Pull request by veronikasamborska1994**

### CODE REVIEW

**Comment by veronikasamborska1994** [comment]
hey @Marigold :) I think this should be good now and the CPI import step is now in snapshot. I am still getting a unit test which I'll fix now but if you have a suggestion for how to best avoid it let me know :) and thanks!! 
it's caused by me accessing the download_url in snapshot:

```
    url = snap.metadata.origin.url_download
    df = import_US_cpi_API(url)
```

File: snapshots/wb/2023-09-19/us_cpi.py [hunk 1]
```diff
@@ -1,17 +1,33 @@
...
 
-# change number of years to go back if neccessary (used up to 2000 in 2023)
 
-YEARS = 23
+@click.command()
+@click.option("--upload/--skip-upload", default=True, type=bool, help="Upload dataset to Snapshot")
+def main(upload: bool) -> None:
+    # Create a new snapshot.
+    snap = Snapshot(f"wb/{SNAPSHOT_VERSION}/us_cpi.csv")
+    url = snap.metadata.origin.url_download
```
Comment (Marigold) [inline snapshots/wb/2023-09-19/us_cpi.py:22]: If you add `  # type: ignore` at the end of the line, the error will go away. (Don't be afraid to use it a lot, it's much faster than properly fixing all those type errors)

File: snapshots/wb/2023-09-19/us_cpi.csv.dvc [hunk 2]
```diff
@@ -0,0 +1,33 @@
+meta:
+  origin:
+    # Data product / Snapshot
+    title: Inflation, consumer prices (annual %) - United States (2010 = 100)
+    description: |-
+      Inflation as measured by the consumer price index reflects the annual percentage change in the cost to the average consumer of acquiring a basket of goods and services that may be fixed or changed at specified intervals, such as yearly. The Laspeyres formula is generally used.
+    date_published: latest
+    title_snapshot:
```
Comment (Marigold) [inline snapshots/wb/2023-09-19/us_cpi.csv.dvc:8]: It's empty and could be deleted.

File: snapshots/wb/2023-09-19/us_cpi.csv.dvc [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+    url_main: https://api.worldbank.org/v2/country/us/indicator/FP.CPI.TOTL?
+    url_download: https://api.worldbank.org/v2/country/us/indicator/FP.CPI.TOTL?
+    date_accessed: 2023-09-19
+
+    # License
+    license:
+      url: 
+        https://databank.worldbank.org/reports.aspx?source=2&type=metadata&series=FP.CPI.TOTL.ZG
+      name: CC BY 4.0# License (same as origin.license, for backwards compatibility)
+  license:
```
Comment (Marigold) [inline snapshots/wb/2023-09-19/us_cpi.csv.dvc:25]: You don't need `meta.license`, it's better to have only in `meta.origin.license` (i.e. the one above is sufficient).

**Review by Marigold — APPROVED** [review]
Thanks for fixing it, looks good!

**Pull request merged**